server.go

```diff
@@ -655,7 +655,9 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	if err != nil {
 		return nil, err
 	}
-	s.interceptableSwitch = htlcswitch.NewInterceptableSwitch(s.htlcSwitch)
+	s.interceptableSwitch = htlcswitch.NewInterceptableSwitch(
+		s.htlcSwitch, lncfg.DefaultFinalCltvRejectDelta,
```

Comment:
Another approach here would be to allow callers of the `Intercept` API to specify this delta? That way you can pick your preference. We could validate `> lncfg.DefaultFinalCltvRejectDelta` if we think this is a reasonable minimum.

Reply:
We could do that, but is there a real use case for it?

The reason that I am proposing the specific change in this PR is to simplify the implementation of an external invoice database.

When lnd itself fails htlcs that expire too soon, the interceptor application doesn't need to watch heights. For that, the delta doesn't need to be configurable. The only requirement is that there is a little bit of time for the interceptor to decide what to do before lnd goes to chain. More concretely, it should be possible to hold the htlc for the mpp timeout duration without risking force close.

Reply:
> When lnd itself fails htlcs that expire too soon, the interceptor application doesn't need to watch heights.

What about the case where you intercept htlcs and then hold them for too long? Eg, you're a mobile wallet and you hold the htlc while you open an ad-hoc channel. You need to implement height watcher logic in your interceptor (similair to what lnd already does with locked in htlcs). If the interceptor informed you of a height at which your intercepted htlcs will be dropped, that saves a lot of interceptor code.

Reply:
Not saying we shouldn't also fail htlcs that expire too soon, I just think we may need more than that to cover a wider use-case range. 

> We could do that, but is there a real use case for it?

Possibly not, non-blocking on making it configurable. Can always come back and add a config option if its wanted.

Reply:
>What about the case where you intercept htlcs and then hold them for too long? Eg, you're a mobile wallet and you hold the htlc while you open an ad-hoc channel. 

Yes, I was just talking about my own use case and that this PR is what I minimally need.